### PR TITLE
Fix Javadoc

### PR DESCRIPTION
**Pull request by vbauer**
It is just a small part of fixes in Javadoc (and corresponding code, usually methods signatures). The initial idea was to cleanup everything to remove Maven profile `default-jdk18-profile`, but there are a lot of javadoc issues.

I'm not sure that I will fix all of them, so it will be useful to hear some feedback about this work. Is it useful or not? I can continue a little bit.

### CODE REVIEW

**Comment by azurecla** [comment]
Hi __@vbauer__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>You've already signed the contribution license agreement. Thanks!</span>
        <span>We will now validate the agreement and then real humans will evaluate your PR.</span>

TTYL, AZPRBOT;

File: src/main/java/microsoft/exchange/webservices/data/autodiscover/request/AutodiscoverRequest.java [hunk 1]
```diff
@@ -534,8 +524,8 @@ protected void writeSoapRequest(URI requestUrl,
    * Write extra headers.
    *
    * @param writer The writer
-   * @throws ServiceXmlSerializationException
-   * @throws javax.xml.stream.XMLStreamException
+   * @throws ServiceXmlSerializationException the service xml serialization exception
+   * @throws XMLStreamException the xML stream exception
```
Comment (vboctor) [inline src/main/java/microsoft/exchange/webservices/data/autodiscover/request/AutodiscoverRequest.java]: xML -> xml or Xml.

**Comment by vboctor** [comment]
:+1: There are somes changes that are not javadoc related, but the changes all look good. Thanks @vbauer

File: src/main/java/microsoft/exchange/webservices/data/autodiscover/request/AutodiscoverRequest.java (hunk 1 above)
Comment (vbauer) [inline src/main/java/microsoft/exchange/webservices/data/autodiscover/request/AutodiscoverRequest.java]: Yes, I'll fix it. It was just a copy-paste from other places (to be in one way). :)

**Comment by vbauer** [comment]
Ok, so I will continue to work on javadocs.

**Comment by serious6** [comment]
:+1:

**Comment by vbauer** [comment]
I've just pushed additional fixes, but it looks like never-ending story. :) I just want to notify, that I will not be able 16.04-01.05 (vacation). Maybe it will be better to merge current changes to prevent obsolescence. What do you think?

**Comment by vboctor** [comment]
It makes sense to split into multiple change batches and also get in to avoid merge conflicts. I haven't checked latest changes but the first batch looked good.

**Comment by vbauer** [comment]
I've just updated patches and merged conflicts.

**Comment by vbauer** [comment]
I updated patch (squashed it) and merged conflicts.

**Comment by serious6** [comment]
@vbauer hmm maybe your master is not up-to-date? There are still 2 other commits.

**Comment by vbauer** [comment]
@serious6 Fixed.

**Comment by serious6** [comment]
:+1: thanks for the contribution.

**Comment by vbauer** [comment]
Thank you for merge!

**Pull request merged**